packages/components/src/components/as-stacked-bar-widget/as-stacked-bar-widget.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../../../styles/src/core/_dev';
+@import '../../../../styles/src/tooltip/tooltip';
+
+as-stacked-bar-widget {
+  --stacked-bar-widget-background-color: var(--widget-background-color, #{$color-ui-01});
+
+  display: block;
+  width: 100%;
+  max-height: 100%;
+  padding: 16px;
```

Comment:
Can we use get-spacing(4) here?

Also, if we add these 16px we should add them to the Histogram as well.

Right now, if you add both on a sidebar they are not aligned at all, as it shows on the acceptance example